rasa/core/actions/forms.py

```diff
@@ -428,6 +441,12 @@ def _temporary_tracker(
             slots=domain.slots,
         )
 
+    async def user_rejected_manually(self, validation_events: List[Event]) -> bool:
+        """Returns True if user return ActionExecutionRejected event, else False."""
```

Comment:
`if user return ActionExecutionRejected event` seems a bit hard to understand. Could please elaborate a bit in the docstring?